corehq/motech/fhir/const.py

```diff
@@ -2,3 +2,8 @@
 FHIR_VERSIONS = [
     (FHIR_VERSION_4_0_1, 'R4'),
 ]
+SUPPORTED_FHIR_RESOURCE_TYPES = [
```

Comment:
Will this list always be small, for usability, or will it grow to include all FHIR resource types? (i.e. the return value of [FHIRResourceType.get_names()](https://github.com/dimagi/commcare-hq/pull/29100/files#diff-433e7032abf02a0c78231568078bff8181998607bf011edb22ea4b1d3018041fR59))

Reply:
For the use case that I know of currently, I have limited it to these 3. I don't know if there is any harm in letting this list be all Resource types, so thought of limiting it for now.